jib-cli/src/main/java/com/google/cloud/tools/jib/cli/jar/JarProcessor.java

```diff
@@ -76,7 +81,7 @@ public static JarType determineJarType(Path jarPath) throws IOException {
    * @throws IOException if I/O error occurs when opening the jar file or if temporary directory
    *     provided doesn't exist
    */
-  public static List<FileEntriesLayer> explodeStandardJar(Path jarPath, Path tempDirPath)
+  protected static List<FileEntriesLayer> explodeStandardJar(Path jarPath, Path tempDirPath)
```

Comment:
Perhaps this can be package-private?
```suggestion
  static List<FileEntriesLayer> explodeStandardJar(Path jarPath, Path tempDirPath)
```
And if this needs to be package-private only to be able to call from tests (i.e., it should have been `private` if not for the tests), add `@VisibleForTesting`.